### PR TITLE
docs: Raspbery Pi / Docker / zwave2js container how-to for consideration.

### PR DESCRIPTION
**Pull request by scyto**
I made this from the notes i did for myself.

I would like to propose this to help bootstrap folks less familiar with zwavejs2mqtt.

1) i am not sure all the settings are 'best' but they work on mine, happy to take suggested edits or clarifications on how to help folks choose settings - this was hardest part when i started, i had to scour multiple web how to.

2) do you think i should expand to include how to put MQTT on hass in  supervisor - for now it is an assumption they have mosquito via docker, hass or some other mechanism

Or if its not useful feel free to reject the PR :-)  or suggest edits / improvements etc

### CODE REVIEW

**Comment by robertsLando** [comment]
Hi @scyto , thanks for your pr! :) I would suggest you to move it to a discussion instead of docs, so users can find it easily. There are many things to fix in this PR as the language used is like a personal note more then docs, but I know some users may find it useful so move it to a discussion ok?

File: docs/getting-started/rpi-docker-howto.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ 	sudo apt install python3 
+ 	sudo apt install python3.pip
+ 	sudo pip3 -v install docker-compose  (is the -v really needed?)
+
+# Configure zwavejs2mqtt container
+
+	cd ~
+ 	mkdir zwavejs2mqtt
+	cd zwavejs2mqtt
+	curl -fsSL https://raw.githubusercontent.com/OpenZwave/Zwave2Mqtt/master/docker/docker-compose.yml -o docker-compose.yml
```
Comment (roflmao) [inline docs/getting-started/rpi-docker-howto.md:51]: zwave2mqtt is the old project

Comment (robertsLando) [reply docs/getting-started/rpi-docker-howto.md:51]: lol missed that link hahah

Comment (scyto) [reply docs/getting-started/rpi-docker-howto.md:51]: doh, cut and paste

**Comment by scyto** [comment]
never seen or used discussions before, oh wait they are beta? cool will figure that out

**Comment by scyto** [comment]
done, added as show and tell and fixed the curl link :-)

**Pull request closed without merging**